### PR TITLE
Tidy up singular vs. plural session keys.

### PR DESCRIPTION
**Pull request by davidben**
How's this look?

### CODE REVIEW

File: draft-ietf-tls-tls13.md [hunk 1]
```diff
@@ -4878,7 +4878,7 @@ is intended to provide both one-way authenticated (server-only) and
 mutually authenticated (client and server) functionality. At the completion
 of the handshake, each side outputs its view of the following values:
 
-- A "session key" (collectively the various secrets derived from
+- A set of "session keys" (collectively the various secrets derived from
```
Comment (martinthomson) [inline draft-ietf-tls-tls13.md]: drop "collectively" now

File: draft-ietf-tls-tls13.md [hunk 2]
```diff
@@ -4906,7 +4906,7 @@ Peer Authentication.
 identity. If the client is authenticated, the server's view of the
 peer identity should match the client's identity.
 
-Uniqueness of the session key:
+Uniqueness of the session keys:
```
Comment (martinthomson) [inline draft-ietf-tls-tls13.md]: The need for uniqueness applies not to the set, but to every individual key in that set.

File: draft-ietf-tls-tls13.md (hunk 1 above)
Comment (davidben) [inline draft-ietf-tls-tls13.md]: Done.

File: draft-ietf-tls-tls13.md (hunk 2 above)
Comment (davidben) [inline draft-ietf-tls-tls13.md]: Added an extra sentence.

**Comment by davidben** [comment]
PS: I think this is obligatory. :-)

there-are-four-keys

**Comment by martinthomson** [comment]
I was looking for that.

**Pull request merged**